EIPS/eip-draft_erc20_approval_expiration.md

```diff
@@ -0,0 +1,187 @@
+---
+eip: <to be assigned>
+title: Approval Expirations for ERC20 Tokens
+description: This standard introduces a standardized way for users to set expiration limits on ERC-20 token approvals.
+author: Go+ Security <@GoPlusSecurity>, Jeff Hall <@JeffHal25193696>, Xavi <@XaaaaavitheFool>, Turan Vural <@turanzv>
```

Comment:
Github names need to be wrapped in parentheses like this `lightclient (@lightclient)`.

Reply:
fixing